src/components/Breadcrumb/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactElement } from 'react';
+import { BrowserRouter, Link } from 'react-router-dom';
+import clsx from 'clsx';
+import Typography from '../Typography';
+import { Props } from './types';
+
+
+function Breadcrumb({
+    className,
+    divider,
+    crumbs,
+}: Props): ReactElement {
+
+
+    const crumbCount = crumbs.length - 1;
+    const classes = clsx(
+        'brew-Breadcrumb',
+        className,
+    );
+
+
+    return (
+        <div className={classes}>
+            {crumbs.map((crumb, index) => {
+                const notLast = crumbCount !== index;
+                return (
+                    <BrowserRouter key={index}>
+                        <Typography variant="body2">
+                            {crumb.url && notLast ?
+                                <Link className="brew-Breadcrumb--link" to={crumb.url}>{crumb.title}</Link> :
+                                <Typography variant="body2">{crumb.title}</Typography>
+                            }
```

Comment:
Should the ternary lines be more like this

```
crumb.url && notLast
    ? true
    : false
```
I feel like it's a little more readable

Reply:
I second this.